packages/s/sol2/xmake.lua

```diff
@@ -28,8 +29,7 @@ package("sol2")
     end)
 
     on_install(function (package)
-        local configs = {}
-        import("package.tools.cmake").install(package, configs)
+        os.cp("include/sol",package:installdir("include"))
```

Comment:
please revert it, cmake.install installs an additional *.cmake import file, which is useful for other packages that depend on sol2 and helps to find sol2 in other packages using cmake's find_package.